multimodal/setup.py

```diff
@@ -20,43 +20,41 @@
 submodule = "multimodal"
 install_requires = [
     # version ranges added in ag.get_dependency_version_ranges()
-    "numpy",
-    "scipy",
-    "pandas",
-    "scikit-learn",
-    "Pillow",
-    "tqdm",
+    "numpy",  # version range defined in `core/_setup_utils.py`
+    "scipy",  # version range defined in `core/_setup_utils.py`
+    "pandas",  # version range defined in `core/_setup_utils.py`
+    "scikit-learn",  # version range defined in `core/_setup_utils.py`
+    "Pillow",  # version range defined in `core/_setup_utils.py`
+    "tqdm",  # version range defined in `core/_setup_utils.py`
     "boto3",
     "requests",
-    "jsonschema<=4.8.0",
-    "seqeval<=1.2.2",
-    "evaluate<=0.3.0",
-    "accelerate>=0.9,<0.16",
+    "jsonschema<4.18",
+    "seqeval<1.2.3",
+    "evaluate<0.4.0",
+    "accelerate>=0.9,<0.17",
     "timm<0.7.0",
     "torch>=1.9,<1.14",
     "torchvision<0.15.0",
     "fairscale>=0.4.5,<0.4.14",
     "scikit-image>=0.19.1,<0.20.0",
-    "smart_open>=5.2.1,<5.3.0",
     "pytorch_lightning>=1.8.0,<1.10.0",
-    "text-unidecode<=1.3",
+    "text-unidecode<1.4",
     "torchmetrics>=0.8.0,<0.9.0",
```

Comment:
Latest version for this is 0.11.1, could the cap be lifted?

Reply:
@zhiqiangdon Can we list the cap of torchmetrics?

Reply:
We can't loose the cap of torchmetrics for now. We need to make some changes in our data preprocessing to loos the cap.